### ISSUE TITLE
Disabling "Disable Firefox Studies" doesn't enable it

### ISSUE THREAD

**Issue by Chartman123** [mozilla/policy-templates#368]
We faced the add-ons problem and wanted to fix it by enabling the Firefox Telemetry and Studies. Unfortunately disabling the respective GPOs only enables the Telemetry.

For the Studies, it only enables the checkbox but doesnt check it. At least on 60.4.0esr which we're currently using.

Is this also true for newer versions?

**Comment by mkaply** [comment]
Studies and Telemetry are enabled by default so just not disabling them should have been enough.

Were they disabled before?

**Comment by Chartman123** [comment]
Yes, both policy items were enabled before, so that the functions were disabled. Switching them to disable only enabled the Telemetry, but not the Studies.
